client/landing/stepper/declarative-flow/internals/steps-repository/components/calendy-widget/index.tsx

```diff
@@ -60,10 +72,16 @@ const CalendlyWidget: React.FC< CalendlyWidgetProps > = ( { url, id, prefill, on
 
 			const element = document.getElementById( widgetId );
 			if ( element ) {
+				const queryParams = new URLSearchParams();
+
+				queryParams.set( 'hide_landing_page_details', hideLandingPageDetails ? '1' : '0' );
+				queryParams.set( 'hide_event_type_details', hideEventTypeDetails ? '1' : '0' );
+				queryParams.set( 'hide_gdpr_banner', hideGdprBanner ? '1' : '0' );
+
 				// Clear out the container div when props change.
 				element.innerHTML = '';
 				window.Calendly.initInlineWidget( {
```

Comment:
I see we're using initInlineWidget here, by looking into the embed options [here](https://help.calendly.com/hc/en-us/articles/223147027-Embed-options-overview#embed-options-overview-0-0), I'm wondering if the popup text one is more suitable for us? Could you help me understand the reason of using `initInlineWidget` here instead of `initPopupWidget`? Is it because we don't have the control over the description text?